stdlib/std/debug.tact

```diff
@@ -8,7 +8,4 @@ native nativeThrowWhen(code: Int, condition: Bool);
 native throw(code: Int);
 
 @name(throw_unless)
-native nativeThrowUnless(code: Int, condition: Bool);
-
-@name(dump_stack)
-native dumpStack();
```

Comment:
Hmm, we cannot remove functions from stdlib without breaking backwards compatibility. Is there a way to keep `dumpStack` here?

Reply:
How is this breaking backwards compatibility? The function is still available the same way as before, it’s just defined natively in language rather than in stdlib.

Reply:
Oh, looks like throwUnless was removed somehow. Sorry for that, probably I selected that line accidentally.

Reply:
No, no. `throwUnless` is fine. I mean before this change the user needed to explicitly import `std/debug.tact` and only then `dumpStack` would be available. To make it backwards compatible we'd need to make `dumpStack` (defined internally) available only after the proper input.

Reply:
Ah, all of the modules in the `std/` directory are automatically imported. So, this does not change anything. Sorry, my bad

Reply:
Having it in stdlib will make line printing impossible. Maybe we can just leave dumpStack as it was before? Anyway it is a low-level thing that should not be used commonly by devs in Tact.

Reply:
I mean your initial approach is the right one. Let's add it in the language and remove from stdlib (it actually won't change anything). With Tact 1.2 I cannot re-use the `dumpStack` name:

```
fun dumpStack() {
    dump("foo");
}
```

will result in this error message

```
/main.tact:1:1: Static function dumpStack already exists 
Line 1, col 1: 
> 1 | fun dumpStack() { 
      ^~~~~~~~~~~~~~~~~ 
  2 |     dump("foo"); 
```

I was worried about this